docs/ACME.md

```diff
@@ -71,14 +71,22 @@ location /.well-known/acme-challenge {
 }
 ```
 
-For Apache, add the following to your existing webserver config::
+For Apache, add the following to your existing webserver config:
 
 ```
 ProxyPass /.well-known/acme-challenge http://localhost:8009/.well-known/acme-challenge
 ```
 
 Make sure to restart/reload your webserver after making changes.
 
+Finally, make the relevant changes in `homeserver.yaml` to enable ACME support:
+
+```
+acme:
+    enabled: true
+    port: 8009
+```
+
```

Comment:
... and restart synapse

Reply:
It tells you to at the end of the instructions. Is that too far away that people will think to finish at this point?

Reply:
ah ISWYM. hopefully they will figure it out.